src/Microsoft.ML.Transforms/doc.xml

```diff
@@ -39,14 +41,344 @@
         for Ind they are concatenated and for Bag they are added.
         When the source column is a singleton, the Ind and Bag options are identical.</para>
       </remarks>
+    </member>
+    <example name="CategoricalOneHotVectorizer">
       <example>
-        An example of how to add the CategoricalOneHotVectorizer transform to a pipeline with two text column 
+        An example of how to add the CategoricalOneHotVectorizer transform to a pipeline with two text column
         features named &quot;Text1&quot; and &quot;Text2&quot;.
-        <code>
+        <code language="csharp">
           pipeline.Add(new CategoricalOneHotVectorizer(&quot;Text1&quot;, &quot;Text1&quot;));
         </code>
       </example>
+    </example>
+
+    <member name="CountFeatureSelection">
+      <summary>
+        Selects the slots for which the count of non-default values is greater than or equal to a threshold.
+      </summary>
+      <remarks>
+        <para>
+          This transform uses a set of aggregators to count the number of non-default values for each slot and
+          instantiates a <see cref="Microsoft.ML.Runtime.Data.DropSlotsTransform"/> to actually drop the slots.
+          This transform is useful when applied together with a <see cref="T:Microsoft.ML.CategoricalHashOneHotVectorizer"/>. 
+          The count feature selection can remove those features generated by the hash transform that have no data in the examples.
+        </para>
+      </remarks>
+    </member>
+    <example name="CountFeatureSelection">
+       <example>
+        <code language="csharp">
+          pipeline.Add(new FeatureSelectorByCount() { Column = new[]{ &quot;Feature1&quot; }, Count = 2 });
+        </code>
+      </example>
+    </example>
+
+    <member name="MutualInformationFeatureSelection">
+      <summary>
+        Selects the top k slots across all specified columns ordered by their mutual information with the label column.
+      </summary>
+      <remarks>
+        <para>
+          The mutual information of two random variables X and Y is a measure of the mutual dependence between the variables.
+          Formally, the mutual information can be written as:
+        </para>
+          <para>I(X;Y) = E[log(p(x,y)) - log(p(x)) - log(p(y))]</para>
+        <para>where the expectation is taken over the joint distribution of X and Y. 
+        Here p(x,y) is the joint probability density function of X and Y, p(x) and p(y) are the marginal probability density functions of X and Y respectively. 
+        In general, a higher mutual information between the dependent variable (or label) and an independent variable (or feature) means 
+        that the label has higher mutual dependence over that feature.
+        The mutual information feature selection mode selects the features based on the mutual information. 
```

Comment:
This sounds like a tautology to me.

Reply:
it's not adding to the understanding either. Removing it. 

---
In reply to: [203200715](https://github.com/dotnet/machinelearning/pull/529#discussion_r203200715) [](ancestors = 203200715)